### DISCUSSION TITLE
Getting "The media could not be loaded, either because the server or network failed or because the format is not supported."

### DISCUSSION THREAD

**Discussion by simondsmason** [blakeblackshear/frigate#4822] [Ask A Question]
I have started getting this a lot on some of my cameras.  I looked through open issues etc. and found one that was closest to my issue.  I posted on it - https://github.com/blakeblackshear/frigate/issues/4818 - but then realized after doing it that this issue was closed and assumed resolved.

So my question is more about etiquette.  Should I start a new issue or did I do the right thing to post on the existing issue?  Thanks.

**Comment by NickM-27** [comment]
That error message is very generic and there's many different reasons that error message can happen. We have many open issues for this already so if it seems related then commenting on open issues is fine. When in doubt create your own

In your case where it's specific to cameras you likely have corrupt segments that were saved. This is already fixed in the next version but for now I'll see if I can find the issue with instructions to fix.

**Comment by simondsmason** [comment]
Thanks @NickM-27.  I looked at other threads but didn't see anything that was still open.  I will keep looking.

**Reply by NickM-27** [reply]
I would recommend looking at this one https://github.com/blakeblackshear/frigate/issues/4233#issuecomment-1362015498

**Comment by jonadis** [comment]
Have a look at the docker container logs:
```
docker logs frigate
```
(where "frigate" is the container name)

**Comment by simondsmason** [comment]
Thanks @jonadis.  Amateur question - What is the best way to read and evaluate these logs?  When I look at them through SSH all I see is it spitting back listings at me - and plenty of errors - but it is not easy to review and understand here.  I did search for this but couldn't see any pointers - which I thought was strange as I imagine this is something people have to do all of the time.  I must be missing something here?  Thanks.

**Comment by jonadis** [comment]
One way to go about it would be to "follow" the logs (watch them live) with the -f flag
```
docker logs frigate -f
```
and then click on an event (or do whatever it is that you're doing that generates the "the media could not be loaded..." message) and watch the corresponding live output in the logs

Another thing you could do is to pipe the log output to less
```
docker logs frigate 2>&1 | less
```
then you can use the arrow keys to move up and down through the logs ... and the forward slash (/) to search for a string.  See the less manpage for more information

**Comment by simondsmason** [comment]
@jonadis Thanks - It definitely seems to be happening on specific cameras - haven't been able to log it yet.  However, if the fix mentioned above might take care of this I may sit tight and just disabling clips on these cameras for now.

**Comment by simondsmason** [comment]
@NickM-27 I have been able to confirm that it only happens on the Reolink cameras that I have that don’t support iframe.  The cameras where this setting is available and configured do not have any issues playing back clips.  I have been monitoring the logs but haven’t found anything yet to explain why my other Reolinks are having issues.  Assuming this is the fix mentioned above, when will that be released approximately?  Thanks.

**Reply by NickM-27** [reply]
No idea, about to go for beta3
